File: bioinformatics/countwithpseudocounts.py
# Input:  A set of kmers Motifs
# Output: CountWithPseudocounts(Motifs)
def Count(Motifs):
    d = {}
    chars = ['A', 'C', 'G', 'T']
    for i in chars:
        d[i] = [1] * len(Motifs[0])
    for col in range(len(Motifs[0])):
        for row in range(len(Motifs)):
            d[Motifs[row][col]][col] += 1
    return d

def CountWithPseudocounts(Motifs):
    t = len(Motifs)
    k = len(Motifs[0])
    # insert your code here
    return Count(Motifs)

File: bioinformatics/countwithpseudocounts.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    long long t, k;
    cin >> t >> k;
    vector<string> arr(t);
    for (long long i=0;i<t;i++){
        cin >> arr[i];
    }
    map<char, vector<long long>> d;
    vector<long long> temp (k, 0);
    set <char> chars = {'A', 'C', 'G', 'T'};
    for (auto e:chars){
        d[e] = temp;
    }   
    for (long long col = 0;col<k;col++){
        for (long long row = 0;row<t;row++){
            
        }
    }
}